docs/QuickstartBuildAgent.mdx

```diff
@@ -0,0 +1,154 @@
+---
+id: QuickstartBuildAgent
+title: Build an Agent
+sidebar_label: Build an Agent to Capture Data
```

Comment:
can you make this section come after building the flow? That way the flow is already built? Or is the idea to avoid that step? (I forget if this flow already exists in the default tenant)

Reply:
ah nevermind this is explained later.